docs/getting-started/installation.rst

```diff
@@ -78,19 +78,19 @@ Scipp is available from `PyPI <https://pypi.org/>`_ via ``pip``:
    pip install scipp
 
 By default, this is only a minimal install without optional dependencies.
-To install components for additional submodules, use the ``all`` extras of the module:
+To install all optional dependencies, including libraries for interactive plotting in Jupyter, use:
 
 .. code-block:: sh
 
    pip install scipp[all]
 
-This will install everything that is needed, with the exception of plotting libraries for interactive use in Jupyter.
-These are available in the ``interactive`` extra.
-We recommend combining this with ``all``:
+
+You can also leave out the interactive tools and bring in only functional optional dependencies,
+such as ``h5py`` and ``scipy``, use ``extra`` instead of ``all``:
```

Comment:
It would still be useful to mention `interactive` as a possibility.

Reply:
Is it really worth it? Would you really install `interactive` without `extra`? I thought if you've decided to install all the interactive stuff, you probably don't mind the few extra packages in `extra`?

Leaving it out has the advantage of making this page a little less long/confusing.

Reply:
But then we should remove the `interactive` extra altogether?

Reply:
We could. What would be the syntax in `pyproject` then? Not sure I know how...

Reply:
I would think it is just
```toml
all = [
  "scipp[extra]",
  "h5py",
  ...
]
```